docs/README.md

```diff
@@ -2,6 +2,7 @@
 
 - [Introduction](introduction.md)
 - [Quick Start](quickstart.md)
+- [Overview](overview.md)
```

Comment:
Is it too much to have "Introduction" and "Overview"? It feels like ski trail names: "Upper Introduction", "Lower Introduction", "Middle Overview", "Lower-mid Introduction"...

Reply:
Perhaps. My original name for it was Concepts, but the section itself broke naturally into Architecture and Concepts, so that didn't seem right. Perhaps I change this to `concepts.md` and replace the Concepts header in the doc with... "Components"? "Experiments," since that's the top-level concept?

Reply:
@jgustie Should I revise this or merge as-is?

Reply:
It's fine as is for now.